database/src/migrations/release.0.22/vw_survey_status.sql

```diff
@@ -6,12 +6,12 @@ with not_published as (select os.survey_id, max(ss.submission_status_id) as subm
 		where os2.survey_id = os2.survey_id
 		and ss2.occurrence_submission_id = os2.occurrence_submission_id
 		and sst.submission_status_type_id = ss2.submission_status_type_id
-		and sst.name = api_get_character_system_constant('SURVEY_STATE_PUBLISHED')
+		and sst.name = api_get_character_system_constant('OCCURRENCE_SUBMISSION_STATE_PUBLISHED')

```

Comment:
Not this specific line, but the whole function): Can you add some comments/descriptions about what this function is doing and anything else relevant? Whatever you think would help someone in future understand/maintain this.  Same goes for any other functions as well really.

Reply:
will do.